bangazonapi/views/paymenttype.py

```diff
@@ -81,10 +81,10 @@ def list(self, request):
         """Handle GET requests to payment type resource"""
         payment_types = Payment.objects.all()
 
-        customer_id = self.request.query_params.get('customer', None)
+        customer = request.auth.user
```

Comment:
Take another look at your request.auth.user. What's the user's id? Look at the customer related to that user. What is the customer's id? HINT: They are not the same.